AnkiDroid/src/main/java/com/ichi2/anki/AnkiDroidApp.java

```diff
@@ -157,7 +157,7 @@
      * collections being upgraded to (or after) this version must run an integrity check as it will contain fixes that
      * all collections should have.
      */
-    public static final int CHECK_DB_AT_VERSION = 20900148;
+    public static final int CHECK_DB_AT_VERSION = 21000150;
```

Comment:
:man_facepalming: just did 50, next is 51 sorry
```suggestion
    public static final int CHECK_DB_AT_VERSION = 21000151;
```